bookkeeper-server/src/test/java/org/apache/bookkeeper/client/TestFencing.java

```diff
@@ -334,6 +334,46 @@ public void testFencingInteractionWithBookieRecovery2() throws Exception {
         writelh.close();
     }
 
+    /**
+     * create a ledger and write entries.
+     * sleep a bookie
+     * Ensure that fencing proceeds even with the bookie sleeping
+     */
+    @Test
+    public void testFencingWithHungBookie() throws Exception {
+        System.setProperty("digestType", digestType.toString());
```

Comment:
What is this for?

Reply:
@athanatos do you mind answering this comment? otherwise this change is ready to go.

Reply:
@ivankelly Apologies, it's left over from the test I adapted.  I'll clean it up.